tests/TestCase/Database/Type/DateTimeTypeTest.php

```diff
@@ -203,7 +203,78 @@ public function marshalProvider(): array
             'src/I18n/Time.php',
         ]);
 
-        $data = [
+        $data = $this->dateTimeArray();
+
+        Configure::delete('Error.ignoredDeprecationPaths');
+
+        return $data;
+    }
+
+    /**
+     * Data provider for marshal()
+     *
+     * @return array
+     */
+    public function marshalWithTimezoneProvider(): array
+    {
+        Configure::write('Error.ignoredDeprecationPaths', [
+            'src/I18n/Time.php',
+        ]);
+
+        $defaultTimezone = date_default_timezone_get();
+        date_default_timezone_set('Europe/Vienna');
+        $data = $this->dateTimeArray();
+        Configure::delete('Error.ignoredDeprecationPaths');
+        date_default_timezone_set($defaultTimezone);
```

Comment:
This needs to happen in tearDown() in case of errors.

Reply:
refactored that to be done in setUp() and tearDown()